training/vision/components/instance_segmentation/spec.yaml

```diff
@@ -70,21 +206,47 @@ outputs:
 
 code: ../src
  
-environment: azureml:AzureML-AutoML-DNN-Vision-GPU:98
+environment: azureml:AzureML-AutoML-DNN-Vision-GPU:107
```

Comment:
is there a way to always take latest env ? (like if we remove the version no). Would it help if we always take latest env ?

Reply:
updated the env to use the latest always... here is the run - https://ml.azure.com/experiments/id/f6a41276-fe9e-4be4-875f-908db3add077/runs/06dee5bc-e00d-4674-bfa8-eadb234512c1?wsid=/subscriptions/381b38e9-9840-4719-a5a0-61d9585e1e91/resourceGroups/automlimage_eastus2_rg/providers/Microsoft.MachineLearningServices/workspaces/risha-ws-eastus2&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#